embeddings.go

```diff
@@ -16,7 +16,7 @@ var ErrVectorLengthMismatch = errors.New("vector length mismatch")
 type EmbeddingModel string
 
 const (
-	// Deprecated: The following block will be shut down on January 04, 2024. Use text-embedding-ada-002 instead.
+	// Deprecated: The following block is shut down. Use text-embedding-ada-002 instead.
```

Comment:
IMO these should just be removed now that they are shutdown, but wasn't sure that would cause issues